test/lib/helpers/broker_event_transformation_test_with_source.go

```diff
@@ -502,26 +502,26 @@ func createFirstNErrsReceiver(client *lib.Client, firstNErrs int) string {
 
 func createTriggerWithKServiceSubscriber(client *lib.Client,
 	brokerName, kserviceName string,
-	triggerFilter eventingtestresources.TriggerOptionV1Beta1) *eventingv1beta1.Trigger {
+	triggerFilter eventingtestresources.TriggerOptionV1Beta1) *eventingv1.Trigger {
 	client.T.Helper()
 	// Please refer to the graph in the file to check what sample trigger is used for.
 	triggerName := "trigger-broker-" + brokerName
 	return client.Core.CreateTriggerOrFailV1Beta1(
 		triggerName,
-		eventingtestresources.WithBrokerV1Beta1(brokerName),
+		eventingtestresources.WithBrokerV1(brokerName),
 		triggerFilter,
 		eventingtestresources.WithSubscriberServiceRefForTriggerV1Beta1(kserviceName),
```

Comment:
This needs to be changed to [client.Core.CreateTriggerOrFail](https://github.com/knative/eventing/blob/main/test/lib/creation.go#L251), and WithSubscriberServiceRefForTriggerV1